circleguard/gui.py

```diff
@@ -841,11 +842,10 @@ def add_widget(self):
                 widget = EntryWidget(f"{replay.username}'s play with the id {replay.replay_id}", "Delete", replay)
                 widget.pressed_signal.connect(self.remove_replay)
                 self.replays.append(widget)
-                self.result_frame.results.layout.addWidget(widget)
+                self.result_frame.results.layout.insertWidget(0,widget)
```

Comment:
same concern with `insertWidget` here (though not relevant for long, since `VisualizeTab` will get rewritten/moved very soon)

Reply:
See other response. This puts the newest entry at the top